Dockerfile

```diff
@@ -25,6 +25,11 @@ RUN apt-get update && apt-get upgrade -y --no-install-recommends && apt-get inst
     wget \
   && rm -rf /var/lib/apt/lists/*
 
+# Rust
+RUN curl https://sh.rustup.rs -sSf | bash -s -- -y
+ENV PATH="/root/.cargo/bin:${PATH}"
+RUN cargo install cargo-audit
```

Comment:
Add SHA check?

Reply:
Rust provides PGP signatures, I've added these to our dockerfile